app/assets/javascripts/discourse/app/lib/keyboard-shortcuts.js

```diff
@@ -469,6 +470,15 @@ export default {
     composer.focusComposer(event);
   },
 
+  filterSidebar() {
+    const filterInput = document.querySelector(".sidebar-filter input");
```

Comment:
We already have a specific class for this

```suggestion
    const filterInput = document.querySelector(".sidebar-filter__input");
```